bot/src/main/java/com/intern/bot/service/MyBot.java

```diff
@@ -0,0 +1,34 @@
+package com.intern.bot.service;
+
+import java.util.Map;
+import lombok.extern.slf4j.Slf4j;
+import org.telegram.telegrambots.bots.TelegramLongPollingBot;
+import org.telegram.telegrambots.meta.api.objects.Update;
+
+@Slf4j
+public class MyBot extends TelegramLongPollingBot {
+
+  private static final Map<String, String> getenv = System.getenv();
```

Comment:
Use spring `application.properties` file and `@ConfigurationProperties` for custom properties. Check my example: https://github.com/Praytic/discord-reactive-stats-bot/blob/master/src/main/java/org/praytic/discord/statsbot/config/properties/BotProperties.java